shared/bash_remediation_functions/disable_prelink.sh

```diff
@@ -0,0 +1,16 @@
+function disable_prelink {
+	# Disable prelinking and don't even check
+	# whether it is installed.
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
+	else
+		printf '\n%s\n' '# Set PRELINKING=no per security requirements' >> /etc/sysconfig/prelink
+		echo 'PRELINKING=no' >> /etc/sysconfig/prelink
+	fi
+
+	# Undo previous prelink changes to binaries if prelink is available.
+	if test -x /usr/sbin/prelink; then
```

Comment:
I am wonder why you are using the `test` syntax and not the `[` syntax. I know they are the same but the later is more well known to bash devs AFAIK.

Reply:
Probably because `man [` typically won't tell you anything useful, `man test` will go straight to the point.